services/galley/src/Galley/API/Teams.hs

```diff
@@ -908,6 +911,36 @@ setFeatureStatus setter doauth tid status = do
       assertTeamExists tid
   setter tid status
 
+getAllFeaturesH :: UserId ::: TeamId ::: JSON -> Galley Response
+getAllFeaturesH (uid ::: tid ::: _) =
+  json <$> getAllFeatures uid tid
+
+getAllFeatures :: UserId -> TeamId -> Galley Aeson.Value
+getAllFeatures uid tid = do
+  Aeson.object
+    <$> sequence
+      [ w @'Public.TeamFeatureSSO getSSOStatusInternal,
+        w @'Public.TeamFeatureLegalHold getLegalholdStatusInternal,
+        w @'Public.TeamFeatureSearchVisibility getTeamSearchVisibilityAvailableInternal,
+        w @'Public.TeamFeatureValidateSAMLEmails getValidateSAMLEmailsInternal,
+        w @'Public.TeamFeatureDigitalSignatures getDigitalSignaturesInternal,
+        w @'Public.TeamFeatureAppLock getAppLockInternal
+      ]
+  where
+    w ::
```

Comment:
Please avoid one-letter identifiers.  (General rule, it's ok if you have no time to change it here now.)